scripts/enforce_licence.py

```diff
@@ -144,8 +144,8 @@ def TransformCppToJsLicence(licence):
   lines = licence.split('\n')
   output = []
   output.append('/**')
-  for l in lines[1:]:
-    output.append(TransformJsLine(l[2:]))
+  for s in lines[1:]:
```

Comment:
Sorry, can these be `for line in lines...`

Reply:
Done :)